static/js/dashboards-home.js

```diff
@@ -211,27 +211,35 @@ function createSiglensDashboard(inputdbname) {
 
 class btnRenderer {
     init(params) {
-		const starOutlineURL = 'url("../assets/star-outline.svg")';
+        const starOutlineURL = 'url("../assets/star-outline.svg")';
         const starFilledURL = 'url("../assets/star-filled.svg")';
 
         this.eGui = document.createElement('span');
-        this.eGui.innerHTML = `<div id="dashboard-grid-btn">
-             
+        this.eGui.innerHTML = `<div id="dashboard-grid-btn" style="margin-left: 20px;"> <!-- Added margin to move the container to the right -->
                 <button class='btn' id="viewbutton" title="Open dashboard"></button>
                 <button class="btn-simple" id="delbutton" title="Delete dashboard"></button>
                 <button class="btn-duplicate" id="duplicateButton" title="Duplicate dashboard"></button>
-                <button class="star-icon" id="favbutton" title="Mark as favorite" ></button>
-                </div>`;
+                <button class="star-icon" id="favbutton" title="Mark as favorite"></button>
+            </div>`;
+        
         this.dButton = this.eGui.querySelector('.btn-simple');
+        this.dButton.style.marginRight = '25px'; 
         this.duplicateButton = this.eGui.querySelector('.btn-duplicate');
-        this.starIcon=this.eGui.querySelector('.star-icon');
+        this.starIcon = this.eGui.querySelector('.star-icon');
         this.starIcon.style.backgroundImage = favoriteDBsSet.has(params.data.uniqId) ? starFilledURL : starOutlineURL;
 
-        //Disable delete for default dashboards
-		if (defaultDashboardIds.includes(params.data.uniqId)) {
-            this.dButton.disabled = true;
-            this.dButton.title = "Delete disabled";
-			this.dButton.classList.add('default-dashboard-delete'); 
+        //Disable delete for default dashboards and show "Default" label
+        if (defaultDashboardIds.includes(params.data.uniqId)) {
+            const defaultLabel = document.createElement('span');
```

Comment:
1. Add a class to the span, write the style in the CSS file.
2. Align the delete and tag elements to the right.
3. Remove the underline from the default text.
4. Set the background color to `#364fef`.

<img width="200" alt="image" src="https://github.com/siglens/siglens/assets/71771131/eba93985-d3a3-45fd-8561-04b4eec8739e">